providers/common/sql/src/airflow/providers/common/sql/operators/sql.py

```diff
@@ -161,7 +161,12 @@ def get_hook(cls, conn_id: str, hook_params: dict | None = None) -> BaseHook:
         :param hook_params: hook parameters
         :return: default hook for this connection
         """
+        hook_params = hook_params or {}
```

Comment:
This is a good opportunity to add `hook_params` to the doc string to explain what hook params are used for